jcommune-service/src/main/java/org/jtalks/jcommune/service/security/AdministrationGroup.java

```diff
@@ -37,6 +40,17 @@
     private String name;
 
     /**
+     * List of Administration group names
```

Comment:
Pre-defined groups, not Administration groups.. Don't know why this class is named this way..